content/source/docs/cloud/run/install-software.html.md

```diff
@@ -3,19 +3,47 @@ layout: "cloud"
 page_title: "Installing Software in the Run Environment - Runs - Terraform Cloud"
 ---
 
-#  Installing Software in the Run Environment
+# Installing Software in the Run Environment
 
-In rare cases, it might be necessary to install extra software on the Terraform worker, such as a configuration management tool or cloud CLI. This page describes your options for using additional tools within the Terraform worker.
+Terraform relies on provider plugins to manage resources. In most cases, Terraform can automatically download the required plugins, but there are cases where plugins must be managed explicitly.
 
-## Avoid Installing Extra Software
+In rare cases, it might also be necessary to install extra software on the Terraform worker, such as a configuration management tool or cloud CLI.
+
+## Installing Terraform Providers
+
+### Providers Distributed by HashiCorp
+
+Terraform Cloud runs `terraform init` before every plan or apply, which automatically downloads any [providers](/docs/configuration/providers.html) Terraform needs.
+
+Terraform Enterprise instances can automatically install providers as long as they can access releases.hashicorp.com. If that isn't feasible due to security requirements, you can manually install providers. Use [the `terraform-bundle` tool][bundle] to build a custom version of Terraform that includes the necessary providers, and configure your workspaces to use that bundled version.
```

Comment:
Should releases.hashicorp.com be hyperlinked?

Reply:
Not really, but it SHOULD probably be code-span styled. 👍🏼